### DISCUSSION TITLE
keyball44: hold/tap が効かない

### DISCUSSION THREAD

**Discussion by ph-ys** [Yowkees/keyball#228] [Q&A] [answered]
keyball44 を組み立てています。
キー入力についてお伺いさせてください。

問題となっている状況は **REMAP でキーに hold/tap を設定すると hold も tap も入力が効かなくなってしまう**、という状況です。
すべてのキーで試してはおりませんが、 REMAP の keyball44 において濃い目の灰色となっているキーに関しては上記の状況になっています。一方で、**単一のキーコードを担当させた場合は問題なく動作している**ので、半田不備などではなさそうに思います。
firmware は組み立てガイドにもあります[ファームウェアページ](https://remap-keys.app/catalog/tAJ9Htme4oNabUkx4832/firmware)のviaファイルをダウンロードして、QMK Toolbox というソフトウエアで flash しました。 REMAP 上でも flash は試したのですが、下記のようなログがでました（一応 firmware の書き込みは完了しているようですが、最後にエラーがでていたので toolbox を使用しました。また REMAP による flash でも同様の症状の状況でした）。

`Reading bytes from the memory completed.
Writing the firmware finished successfully.
Error: Verification failed: Position:12160 Local:249 MCU:255
`
 
使用している端末のOSはwindowsになります。

状況の改善案をご教授いただけませんでしょうか。
宜しくお願いいたします。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。
私は経験がない症状なのですが、ファームウェア書き込みの問題のように思います。
お使いのProMicroはどちらから入手されたものでしょうか？
RemapでFlashする際のBootloaderの種類を変更して書き込むとエラーが消えたりしないでしょうか？

**Reply by ph-ys** [reply]
お返事ありがとうございます。
本件、何度かProMicroを外したり付け直したり、TRRSケーブルを交換したりした後で再書き込みすることで（ちょっと理由はわからなかったのですが）無事解消されました。。これから使いこなしていきたいと思います。

**Reply by Yowkees** [reply]
原因はわかりませんが、解消されたということで、ひとまず様子を見ていただければ幸いです。
今後ともよろしくお願いいたします。
